examples/adaptiveprecision-blockjacobi/adaptiveprecision-blockjacobi.cpp

```diff
@@ -93,11 +92,11 @@ int main(int argc, char *argv[])
 
     // copy b again
     b->copy_from(host_x.get());
-
+    ValueType reduction_factor = 1e-7;
     auto iter_stop =
         gko::stop::Iteration::build().with_max_iters(10000u).on(exec);
     auto tol_stop = gko::stop::ResidualNormReduction<ValueType>::build()
-                        .with_reduction_factor(static_cast<ValueType>(1e-12))
+                        .with_reduction_factor(reduction_factor)
```

Comment:
If this needs to be larger than `reduction_factor`, then maybe:
```suggestion
                        .with_reduction_factor(reduction_factor * 10e2)
```

Reply:
Sorry, but I dont understand why it should be `reduction_factor * 10e2` . It is for the CG solver. So, I think a normal `reduction_factor` should be valid.

Reply:
My bad, I thought `reduction_factor` was used for something else here as well, but apparently, it is only used for the stopping criterion.